python/MDSplus/tests/_common.py

```diff
@@ -200,13 +200,19 @@ def runWithObjgraph(cls, tests=None):
 
     @classmethod
     def main(cls):
+        if 'TEST_INDEX' in os.environ:
+            cls.index = int(os.environ['TEST_INDEX'])
```

Comment:
Is `cls.index` just a hook for the future?   (I don't see where it is used.).  Regardless, I do like this change.

Reply:
Yeah, I think so. It was present in the testing framework, and seemed like it would be used for this exact sort of thing, so I decided to re-use it instead of make another one.

Reply:
A-OK.  I concur that it is good to have `cls.index` as it might be useful in the future.